src/features/pages/acuity/acuity-feature-guide/data.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { TAcuityFeatures } from '../../../components/molecules/image-text-boxes/types'
+import { deriv_acuity_eu_download, deriv_acuity_row_download } from 'common/constants'
+
+export const contentData: TAcuityFeatures = {
+    mobileFlexDirection: 'col-reverse',
+    data: [
+        {
+            id: 0,
+            subtitle:
+                '_t_Download the Acuity suite of trading tools to your MT5 desktop terminal and get access to an array of valuable market insights and trading opportunities._t_',
+            image: (
+                <StaticImage
```

Comment:
I am not sure if we use **StaticImage** for the new approach but in case we don't you could also check out the new Image component from `import Image from 'features/components/atoms/image'`

Reply:
They will replace later @konstantinosG-deriv on iFrame/widget

Reply:
so @NikitK-deriv  and @konstantinosG-deriv  for now I will keep this as it is ?